cloudify_nsx/network/dlr.py

```diff
@@ -32,15 +32,9 @@ def create(**kwargs):
         "dlr_size": {
             "required": True
         },
-        "datacentermoid": {
-            "required": True
-        },
-        "datastoremoid": {
-            "required": True
-        },
-        "resourcepoolid": {
-            "required": True
-        },
+        "datacentermoid": {},
```

Comment:
Yes, make sense. For copy field from inputs  we need to describe filed in validation rules: https://github.com/cloudify-cosmo/cloudify-nsx-plugin/blob/e70dfce40e4bc232464fbfdc5ee47deea2a73066/cloudify_nsx/library/nsx_common.py#L70

So Its fully correct change.